visualization_msgs/CMakeLists.txt

```diff
@@ -27,6 +28,8 @@ set(msg_files
   "msg/Marker.msg"
   "msg/MarkerArray.msg"
   "msg/MenuEntry.msg"
+  "msg/Mesh.msg"
```

Comment:
I think that we should clarify that this is not a Mesh natively but an `EncodedMesh` or `MeshFile` that you're going to have to parse it after reading it. I'm not sure what the best modifier is for meshes.

In a parallel way to we're sending a CompressedImage and you're going to need to uncompress it. Versus just reading the image.

Reply:
Changed to `MeshFile`!